test/unit/tuples.spec.ts

```diff
@@ -119,6 +119,19 @@ test("Tuple Static Method Return Destruct", () => {
     expect(result).toBe(5);
 });
 
+test("Tuple Static Function Property Return Destruct", () => {
+    const result = util.transpileAndExecute(
+        `class Test {
+            /** @tupleReturn */
+            static tuple: () => [number, number, number] = () => [3,5,1];
+        }
+        const [a,b,c] = Test.tuple();
+        return b;`,
+    );
+
+    expect(result).toBe(5);
```

Comment:
I guess it works, but would any of these tests fail if `@tupleReturn` was just ignored?

Reply:
That's a problem with all of these tests (even the previously existing ones). Perhaps we could check the transpiled lua for `unpack`?

Reply:
Yeah that's what I was considering, I don't think we really have a good alternative?